src/samplers/libcall/mod.rs

```diff
@@ -0,0 +1,278 @@
+use async_trait::async_trait;
+
+use std::sync::{Arc, Mutex};
+use std::time::Instant;
+
+use crate::common::bpf::BPF;
+use crate::config::SamplerConfig;
+use crate::samplers::{Common, Sampler};
+
+#[cfg(feature = "bpf")]
+use crate::common::bpf::bpf_hash_char_to_map;
+
+#[cfg(feature = "bpf")]
+use rustcommon_metrics::Statistic;
+
+mod config;
+mod stat;
+
+pub use config::{LibCallConfig, LibFileMap, LibSearchMap};
+pub use stat::LibCallStatistic;
+
+use std::path::Path;
+
+#[allow(dead_code)]
+pub struct LibCall {
```

Comment:
I think you might want to rename this to `Libcall` or consider some other name entirely for both the module and sampler structs?

Typically, we're using the convention that if we use `FooBar` that the config section and module would be called `foo_bar`. You seem to be using `LibCall` and `libcall` instead - which implies it's a single word and should therefore have `Libcall` as its title-case name.

Reply:
We discussed this a bit and I've renamed everything from LibCall -> Usercall.